<?xml version="1.0"?>
<launch>

 <!-- small description about your node -->
 
 <node pkg="cpp_counter" type="cpp_counter_node" name="CppCounterNode" respawn="false" output="screen">
	<rosparam command="load" file="$(find cpp_counter)/config/config_cpp_counter.yaml"/>
</node>

</launch>
